### PR TITLE
Py3 to /docs/README.md

### PR DESCRIPTION
**Pull request by tobes**
Basic Py3 documentation

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -404,6 +406,67 @@ click_info {
 
 ***
 
+
+## <a name="py3"></a>Py3 module helper
+
+Py3 is a special helper object that gets injected into
+py3status modules, providing extra functionality.  A module can access it via
+`self.py3` in its `Py3status` class
```
Comment (pydsigner) [inline doc/README.md]: → A module can access it via the `self.py3` instance attribute of its `Py3status` class.

File: doc/README.md [hunk 2]
```diff
@@ -404,6 +406,67 @@ click_info {
...
+Update a module.  If `module_name` is supplied the module of that
+name is updated.  Otherwise the module calling is updated.
+
+__get_output(module_name)__
+
+Return the output of the named module.  This will be a list.
+
+__trigger_event(module_name, event)__
+
+Trigger the event on named module
```
Comment (pydsigner) [inline doc/README.md]: → Trigger an event event on a named module.

File: doc/README.md [hunk 3]
```diff
@@ -404,6 +406,67 @@ click_info {
...
+
+__notify_user(msg, level='info')__
+
+Send notification to user.
+`level` can be `info`, `error` or `warning`
```
Comment (pydsigner) [inline doc/README.md]: → `level` must be `info`, `error` or `warning`.

File: doc/README.md [hunk 4]
```diff
@@ -404,6 +406,67 @@ click_info {
...
+
+__time_in(seconds=0)__
+
+Returns time seconds in the future.  Helpfull for creating cached_until
```
Comment (pydsigner) [inline doc/README.md]: → Returns the time a given number of seconds into the future.  Helpful for creating the `cached_until` value for the module output.

File: py3status/py3.py [hunk 5]
```diff
@@ -76,7 +76,7 @@ def notify_user(self, msg, level='info'):
 
     def time_in(self, seconds=0):
         """
-        Returns time seconds in the future.  Helpfull for creating cache_until
+        Returns time seconds in the future.  Helpfull for creating cached_until
```
Comment (pydsigner) [inline py3status/py3.py]: Helpfull → Helpful.

**Comment by tobes** [comment]
@pydsigner thanks

File: doc/README.md [hunk 6]
```diff
@@ -404,6 +406,70 @@ click_info {
...
+Update a module.  If `module_name` is supplied the module of that
+name is updated.  Otherwise the module calling is updated.
+
+__get_output(module_name)__
+
+Return the output of the named module.  This will be a list.
+
+__trigger_event(module_name, event)__
+
+Trigger an event event on a named module.
```
Comment (pydsigner) [inline doc/README.md]: "an event event"?

Comment (tobes) [inline doc/README.md]: it's like a super special once in a lifetime event :)

**Comment by ultrabug** [comment]
Thanks guys !

**Pull request merged**